### DISCUSSION TITLE
Get old field value with update row trigger automation

### DISCUSSION THREAD

**Discussion by NicolasFabre81** [Budibase/budibase#15154] [Help]
I need to get the old value of my field in JS.
Doc speaks about {{trigger.oldRow}}.
But neither `$("trigger.oldRow")` nor `$("trigger.oldrow")` work.

What is the syntax ? How comes that we can't see it in bindings ?

EDIT : even `{{trigger.oldRow}}` does not work. `{{trigger}}` only contains the new updated row and not the previous one.
@melohagan 
EDIT2 : maybe solved here @adrinr ?

**Comment by melohagan** [comment]
Hey @NicolasFabre81 Could you link the docs page you mentioned please?

**Reply by NicolasFabre81** [reply]
Here
<img width="715" alt="image" src="https://github.com/user-attachments/assets/0a55434e-1328-4921-a130-2e1cae1d0490">
